backtrader/feeds/ccxt.py

```diff
@@ -52,6 +52,7 @@ class CCXT(DataBase):
     params = (
         ('historical', False),  # only historical download
         ('backfill_start', False),  # do backfilling at the start
+        ('fetch_ohlcv_params', {})
```

Comment:
please, move this to another commit. This has nothing to do with fixing incomplete candles.

Reply:
Hi Ed,
Actually, I think this one is related somewhat. Some exchanges allow you to specify whether partial candles are provided. We can optionally pass these exchange parameters if/when needed using this.

Reply:
I don't see where is this parameter used in this PR. It's just added here and that's it.

BTW, ccxt feed has 'config' parameter. It's also passed to the __init__ of the ccxt store. Can it be used to provide ohlcv parameters?